src/shared/sam/cli/samCliInvokerUtils.ts

```diff
@@ -99,3 +100,13 @@ function startsWithEscapeSequence(text: string, sequences = acceptedSequences):
 function startsWithError(text: string): boolean {
     return text.startsWith('Error:')
 }
+
+export async function addTelemetryEnvVar(options: SpawnOptions | undefined): Promise<SpawnOptions> {
```

Comment:
should this live in chidProcess.ts ?

Reply:
It could be generalized to `addEnvVar` where the key and value are passed as args. But IMO that'd only be more ergonomic if added as a method that mutates the options prior to spawning. Which could be nice?